apps/main/src/lend/lib/apiLending.ts

```diff
@@ -379,10 +382,10 @@ const user = {
             market.userHealth(),
             market.userHealth(false),
             market.userRange(),
-            market.userBands(),
+            !isBeta && market.userBands(),
```

Comment:
It's perhaps a bit outside of the scope of this PR, but you've moved over the functionality of this function to a TanStack query in `@\lend\entities\user-loan-details.ts`. I'm actually planning to delete this code and make sure only the tanstack query is being used. So prob best if we don't touch this function in `apiLending.ts`, and move over the `isBeta` stuff to `user-loan-details.ts`?